robottelo/host_helpers/satellite_mixins.py

```diff
@@ -226,6 +227,15 @@ def get_reported_value(self, report_key):
         assert result.status == 0, 'report failed or key not found'
         return "".join(result.stdout.split(":", 1)[1].split())
 
+    def get_reported_condensed_value(self, report_key):
+        """
+        Runs satellite-maintain report condense and extracts the value for a given key
+        """
+        result = self.cli.SatelliteMaintainReport.condense()
+        assert result.status == 0, 'report failed or key not found'
+        report = "{" + result.stdout.strip().split("{")[1]
```

Comment:
Is this just to separate the actual json from s-m's status updates?

Reply:
yes, I didn't find a better way to disentangle those

Reply:
Could we make it print the stats to a temporary file and then read it from there?

Reply:
fwiw I opened https://github.com/theforeman/foreman_maintain/issues/1025